lib/mathable.rb

```diff
@@ -0,0 +1,8 @@
+module Mathable
+  def average_stuff(arg_1, arg_2)
```

Comment:
Looks good, we'll need to fix the spacing between methods.

Reply:
agreed